flake.nix

```diff
@@ -22,79 +19,16 @@
     papis,
   }:
     flake-utils.lib.eachDefaultSystem (
-      system: let
+      system:
+      let
         pypkgs = pkgs.python3Packages;
```

Comment:
You can delete this line. `pypkgs` isn't used anywherre

Reply:
Removed! Thanks.